src/test_l1_plat_power.c

```diff
@@ -690,7 +692,7 @@ void test_l1_plat_power_negative_PLAT_API_GetWakeupSrc (void)
 {
     gTestID = 12;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
-	pmStatus_t ret;
+    pmStatus_t ret;
     bool enable;
     PWRMGR_WakeupSrcType_t srcType = PWRMGR_WAKEUPSRC_VOICE; // You can choose any valid srcType for this test
```

Comment:
Line 697: in test_l1_plat_power_negative_PLAT_API_GetWakeupSrc API use PWRMGR_WAKEUPSRC_IR instead of PWRMGR_WAKEUPSRC_VOICE

Reply:
Done